.github/workflows/markdown-lint.yml

```diff
@@ -16,11 +16,21 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Copy upstream dependencies
+        run: |
+          curl -L package.json https://raw.githubusercontent.com/mdn/content/main/package.json > package.json
+          curl -L yarn.lock https://raw.githubusercontent.com/mdn/content/main/yarn.lock > yarn.lock
```

Comment:
should this be:

```suggestion
          curl -L https://raw.githubusercontent.com/mdn/content/main/package.json > package.json
          curl -L https://raw.githubusercontent.com/mdn/content/main/yarn.lock > yarn.lock
```

Reply:
Thanks, that got left over from the original try with `-O`. I rebased to fix it in both files